src/applications/financial-status-report/actions/index.js

```diff
@@ -113,5 +113,7 @@ export const downloadPDF = () => {
   return fetch(
     `${environment.API_URL}/v0/financial_status_reports/download_pdf`,
     options,
-  ).catch(err => throw new Error(err));
+  ).catch(err => {
+    throw new Error(err);
```

Comment:
Do want want to log this to Sentry if we are catching it?

Reply:
This question is basically if we want to tract errors use Sentry otherwise just leave it